site/en/blog/chrome-58-media-updates/index.md

```diff
@@ -0,0 +1,263 @@
+tags:
+  - news
+  - media
+  - chrome58
+
+
+
+---
+
+
+- Developers can now [customize media controls](#controlslist) such as the
+  download, fullscreen and remoteplayback buttons.
+- Sites installed using the "Add to Homescreen" flow can [autoplay audio
+  and video in the manifest's scope](#autoplay).
+- Chrome on Android now [pauses autoplaying a muted video when it is invisible](#offscreen).
+- Developers can now access the approximate range of colors supported by Chrome and
+  output devices using the [`color-gamut` Media Query](#colorgamut).
+- When using Media Source Extensions, it's now possible to
+  [switch between encrypted and clear streams].
+
+## Media controls customization
+
+Developers can now customize Chrome's native media controls such as the
+download, fullscreen and [remoteplayback] buttons using the new [ControlsList API].
+
+<div class="float-right">
+<figure>
+{% Img src="image/T4FyVKpzu4WKF1kBNvXepbi08t52/lxOu6rBvfaDJrn2mk8bQ.png", alt="Native media controls in Chrome 58", width="800", height="507" %}
+  <figcaption>
+    <b>Figure 1.</b>
+    Native media controls in Chrome 58
+  </figcaption>
+</figure>
+</div>
+
+This API offers a way to show or hide native media controls that do not make
+sense or are not part of the expected user experience, or only allow a
+limited set of features.
+
+The current implementation for now is blocklist mechanism on native controls
+with the ability to set them directly from HTML content using the new
+attribute `controlsList`. Check out the [official
+sample](https://googlechrome.github.io/samples/media/controlslist.html).
+
+Usage in HTML:
+
+```html
+<video controls <b>controlsList="nofullscreen nodownload noremoteplayback"</b>></video>
+```
+
+Usage in JavaScript:
+
+```js
+    var video = document.querySelector('video');
+    video.controls; // true
+    video.controlsList; // ["nofullscreen", "nodownload", "noremoteplayback"]
+    video.controlsList.remove('noremoteplayback');
+    video.controlsList; // ["nofullscreen", "nodownload"]
+    video.getAttribute('controlsList'); // "nofullscreen nodownload"
+
+    video.controlsList.supports('foo'); // false
+    video.controlsList.supports('noremoteplayback'); // true
+```
+
+[Intent to Ship](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/tFuQd3AcsIQ/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5737006365671424) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=685018)
+
+## Autoplay for Progressive Web Apps added to home screen
+
+Previously, Chrome used to block all `autoplay` with sound on Android without
+exception. This is no longer true. From now on, sites installed using the
+[improved Add to Home Screen] flow are allowed to autoplay audio and video
+served from origins included in the [web app manifest]'s scope without
+restrictions.
+
+```json/3
+{
+  "name": "My Web App",
+  "description": "An awesome app",
+  "scope": "/foo",
+  ...
+}
```

Comment:
The indentation here is part of the sample, not an artifact of the old formatting. If part of the code sample is already at the left margin, leave it alone.